packages/client/src/plugins/fili/serializers/facts.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Description: A serializer for the bard response
+ */
+
+import NaviFactSerializer, { ResponseV1 } from '../../../serializers/facts/interface.js';
+import type { RequestV2, Column } from '../../../request.js';
+import { canonicalizeColumn } from '../../../utils/column.js';
+import NaviFactResponse from '../../../models/navi-fact-response.js';
+import NaviAdapterError, { NaviErrorDetails } from '../../../errors/navi-adapter-error.js';
+import { FactAdapterError } from '../../../adapters/facts/interface.js';
+import { FetchError } from '../../../errors/fetch-error.js';
+import NativeWithCreate, { getInjector } from '../../../models/native-with-create.js';
+import invariant from 'tiny-invariant';
+
+type BardError = {
+  description: string;
+  druidQuery: Record<string, unknown>;
+  reason: string;
+  requestId: string;
+  status: number;
+  statusName: string;
+};
+
+export default class FiliFactsSerializer extends NativeWithCreate implements NaviFactSerializer {
```

Comment:
Is this a good time to start calling these normalizers?

Reply:
Pushing this off for now